packages/timeline-state-resolver-types/src/atem.ts

```diff
@@ -81,11 +81,14 @@ export type SuperSourceBox = {
 }
 
 export interface AtemTransitionSettings {
-	// dip
 	// DVE
 	mix?: {
 		rate: number
 	}
+	dip?: {
+		rate: number
```

Comment:
It would be good to have a comment with the unit, and perhaps the allowed range.